bodhi/server/models/models.py

```diff
@@ -2233,12 +2233,13 @@ def expire(self):
         if self.expired_date is not None:
             return
 
-        koji_session = buildsys.get_session()
-        try:
-            koji_session.untagBuild(self.build.release.override_tag,
-                                    self.build.nvr, strict=True)
-        except Exception, e:
-            log.error('Unable to untag override %s: %s' % (self.build.nvr, e))
+        if datetime.utcnow() >= self.expiration_date:
```

Comment:
I've thought about this some and I don't think the if statement belongs inside the ```expire()``` method. The name "expire" implies that it handles the expire action, not that it does it when needed. The caller of ```expire()``` should carry the burden of deciding when it should be called.

To fix this issue, I think we should find out what is calling ```expire()``` when it should not be called, and stop that from happening. We will also need unit tests.